Alpaca.Markets/Interfaces/IAlpacaStreamingClient.cs

```diff
@@ -7,7 +7,7 @@
 public interface IAlpacaStreamingClient : IStreamingClient
 {
     /// <summary>
-    /// Occurred when new trade update received from stream.
+    /// Invoked when a new trade update is received from the stream.
```

Comment:
Please, review all public events and use the same approach in all cases.

Reply:
I think I've gotten them all. Let me know if you find any I missed.